src/data_hub/dataframe.py

```diff
@@ -19,6 +19,11 @@
 check_for_updates()
 
 
+try:
+    from icecream import ic
+except ImportError:  # Graceful fallback if IceCream isn't installed.
+    ic = lambda *a: None if not a else (a[0] if len(a) == 1 else a)  # noqa
```

Comment:
Could we pull this into a utility file since it seems to be reused?

Reply:
decided to remove it because it's trivial to re-add when debugging, and this will never be neeeded in prod